test/protobuf/decoder_test.exs

```diff
@@ -126,7 +126,7 @@ defmodule Protobuf.Decoder.Test do
       use Protobuf, from: Path.expand("../proto/mumble.proto", __DIR__)
     end
 
-    msg = MumbleProto.Authenticate.new(username: "bitwalker")
+    msg = MumbleProto.MumbleProto.Authenticate.new(username: "bitwalker")
```

Comment:
:+1: this PR includes a few breaking changes and we should think carefully about the users of the library. The use of `inject` has been changed so that it __only__ uses the package names when generating modules (ie not nested at all). I've been wondering if we should re-name the `inject` option to something like `nest_namespaces_in_this_module` (default to true).